sdk/core/Azure.Core/samples/Diagnostics.md

```diff
@@ -86,6 +86,63 @@ using AzureEventSourceListener listener = new AzureEventSourceListener(
 When targeting .NET Standard 2.1, .NET Core 2.2, or newer, you might instead use `e.TimeStamp` to log the time the event was written instead of rendered, like above. It's in UTC format, so if you want to log the local time like in the example call `ToLocaleTime()` first.
 For help diagnosing multi-threading issues, you might also log `e.OSThreadId` which is also available on those same targets.
 
+## ActivitySource support
+
+Azure SDKs released after October 2021 have experimental ActivitySource support. ActivitySource is a simplified way to create and listen to activities added in .NET 5.
+Azure SDKs produce the following kinds of Activities:
+
+- *HTTP calls*: every HTTP call originating from Azure SDKs
+- *client method calls*: for example, `BlobClient.DownloadTo` or `SecretClient.StartDeleteSecret`.
+- *messaging events*: EventHubs and ServiceBus message creation is traced and correlated with its sending, receiving, and processing.
+
+ActivitySource support is experimental.
+We reserve the right to change the shape of Activities in the future, including things like:
+- kinds of operations we track
+- relationships between telemetry spans
+- attributes attached to telemetry spans
```

Comment:
This could be read a bit harshly.  We may want to consider softening to something like:

```suggestion
Because `ActivitySource` support is experimental, the shape of Activities may change in the future without notice.  This includes:

- the kinds of operations that are tracked
- relationships between telemetry spans
- attributes attached to telemetry spans
```